include/can/core/messages.hpp

```diff
@@ -48,13 +48,26 @@ struct Empty : BaseMessage<MId> {
     bool operator==(const Empty& other) const = default;
 };
 
+template <MessageId MId>
+struct Response : BaseMessage<MId> {
+    NodeId node_id;
+
+    void set_node_id(NodeId id) { node_id = id; }
+
+    template <bit_utils::ByteIterator Output, typename Limit>
+    auto serialize_node_id(Output body, Limit limit) const {
```

Comment:
I figured you made this function in order to return an iterator rather than only a size.  But I think this class should still implement a `serialize` method. 

Just for completeness sake.